ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -183,10 +183,10 @@ func TestCreateWithEFSFargate(t *testing.T) {
 	)
 }
 
-func ecsParamsWithFargateEFSVolume() *utils.ECSParams {
+func ecsParamsWithEFSVolume(launchType string) *utils.ECSParams {
 	return &utils.ECSParams{
 		TaskDefinition: utils.EcsTaskDef{
-			ExecutionRole: "arn:aws:iam::123456789012:role/fargate_role",
+			ExecutionRole: fmt.Sprintf("arn:aws:iam::123456789012:role/%s_role", launchType),
```

Comment:
are we validating the role name elsewhere in the tests? wondering why we need to set it with a var

Reply:
You’re right, this doesn’t need to be parameterised. I did this before I realized that service and task were separate packages and though I might be able to share this function between the two test files.